source/speechXml.py

```diff
@@ -112,7 +112,9 @@ def _openTag(self, tag, attrs, empty=False):
 		self._out.append("<%s" % tag)
 		for attr, val in attrs.items():
 			self._out.append(' %s="' % attr)
-			self._out.append(_escapeXml(val))
+			# Attribute values could be ints, floats etc, not just strings.

```

Comment:
Is this in fact a bug fix?

Reply:
Not quite. Originally _escapeXML coerced to unicode. I changed the unicode to str, but then you commented that this could be taken out.
As much as the coersion does need to happen, your comment lead me to research the code a bit more and I then moved the coersion earlier in the code. I didn't think it made sence to be in _escapeXML, as that function's role really should just be to escape xml characters in a string.

Reply:
Thanks for clarifying!